umbraco/Umbraco-CMS
# Fix the introduction tour getting lost if dashboard is not shown

As the title states, the introduction tour will get lost and break if you've had a poke around and have hidden the dashboard before starting the tour, for instance if you start while creating a document type or so on.

I've added a conditional step that's only shown if the dashboard is not, which guides the user to click on the Content section if the dashboard is not shown. I realise that the dashboard could potentially by hidden from the user trying to create a content node, and then it might be a bit misleading to say "click the content button to enter the content section", however clicking the content button when creating content does navigate you back and shows the dashboard, so I think it's still an improvement to have it here.

Now I'm no writer, so I'm not sure if the contents of the step is perfect.

This was initially pointed out in an issue, so this fixes #10570

### Testing
1. Run `npm run build` from the `Umbraco.Web.UI.Client` project
2. Start the tour while the dashboard is not shown
3. Ensure that the tour does not get stuck
